.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 python: 2.7
-install: "pip install flake8"
-script: "flake8 ."
+install:
+  - pip install flake8
+script:
+  - flake8 .
```

Comment:
@stephendonner are you agree with Dave? Because based on your previous comment I understood that I should go back to `flake8`. :)